<?xml version="1.0"?>
<launch>
    <arg name="reverse" default="false"/>
    <node pkg="racecar_navigation" type="graph_planner" respawn="false" name="graph_planner" output="screen">
        <rosparam file="$(find racecar_navigation)/config/graph_planner.yaml" command="load" />

        <param name="upside_down" value="true" if="$(arg reverse)" />
        <param name="spline_file" value="$(find racecar_navigation)/config/graph_path_reverse.txt" if="$(arg reverse)" />
        <param name="spline_file" value="$(find racecar_navigation)/config/graph_path.txt" unless="$(arg reverse)" />
    </node>
</launch>
